src/layouts/docs/menu-contents.ts

```diff
@@ -376,6 +380,10 @@ export const menu: MenuContents = [
             "title": "Auth",
             "href": "/docs/reference/auth"
           },
+          {
+            "title": "Forms",
+            "href": "/docs/reference/forms"
```

Comment:
We should probably include a link in the order app reference section as well

Reply:
It's in both. This line is even the order app one.
It's just the ui only highlights the first one in the menu when you're on the page which is the carrier app link.

Reply:
Ahh, I was looking for it alphabetically on the page... 🤦